www-client/w3m/w3m-0.5.3_p20210102.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools prefix toolchain-funcs vcs-snapshot
+
+MY_P="${P/_p/+git}"
+MY_PV="${PV/_p/+git}"
+
+DESCRIPTION="Text based WWW browser, supports tables and frames"
+HOMEPAGE="https://github.com/tats/w3m"
+SRC_URI="https://github.com/tats/w3m/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="w3m"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="X fbcon gdk-pixbuf gpm imlib l10n_de l10n_ja libressl lynxkeymap nls nntp ssl unicode xface"
+REQUIRED_USE="X? ( ?? ( gdk-pixbuf imlib ) )
+	fbcon? ( ?? ( gdk-pixbuf imlib ) )"
+
+RDEPEND="dev-libs/boehm-gc:=
+	sys-libs/ncurses:0=
+	sys-libs/zlib
+	X? (
+		x11-libs/libXdmcp
+		x11-libs/libXext
+	)
+	gdk-pixbuf? (
+		x11-libs/gdk-pixbuf
+		X? ( x11-libs/gdk-pixbuf-xlib )
+	)
+	!gdk-pixbuf? (
+		imlib? ( media-libs/imlib2[X?] )
+	)
+	gpm? ( sys-libs/gpm )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	xface? ( media-libs/compface )"
+DEPEND="${RDEPEND}
```

Comment:
BDEPEND for pkgconfig -- we call it as a tool, not linked against, so it's BDEPEND

Reply:
Fixed